tst/testrandom.g

```diff
@@ -1,34 +1,38 @@
-randomTest := function(collection, method, checkin...)
-    local test1, test2, test3, test4, test5, test6, localgen, checkmethod;
-    if Length(checkin) = 0 then
-        checkmethod := \in;
-    else
-        checkmethod := checkin[1];
-    fi;
+# Perform a variety of tests on Random generators.
```

Comment:
I still don't understand "Random generators" here! First I thought you meant "random source", but that doesn't make sense. My next guess was "methods for the Random method", but that also doesn't seem to make sense, as the test are not restricted to `Random`.

Right now I think you might mean: "Perform a variety of tests with a given function <method> that is expected to produce/generate random output, and which should allow passing in a random source as optional first argument."

Reply:
I keep using random generator, when I mean source. Will generally clean up.